src/common/xo/index.js

```diff
@@ -1924,7 +1921,22 @@ export const deleteSshKey = key => (
     () => {
       const { preferences } = xo.user
       return _setUserPreferences({
-        sshKeys: filter(preferences && preferences.sshKeys, k => !isEqual(k, key))
+        sshKeys: filter(preferences && preferences.sshKeys, k => k.key !== key)
```

Comment:
Please rebase your branch and use a `resolveId` here since, on the updated version of `SortedTable`, the handlers are called with the items **objects** instead of **IDs**.